src/webots/nodes/utils/WbWorld.cpp

```diff
@@ -79,6 +79,7 @@ WbWorld *WbWorld::instance() {
 
 WbWorld::WbWorld(WbProtoList *protos, WbTokenizer *tokenizer) :
   mWorldLoadingCanceled(false),
+  mResetRequested(false),
```

Comment:
We should probably also initialize this new member variable:
```suggestion
  mResetRequested(false),
  mRestartControllers(false),
```